css/mail.css

```diff
@@ -28,14 +28,18 @@
 
 
 /* SETUP */
-#setup {
```

Comment:
Uh, oh. Also another PR is around for same line. This may later cause conflicts on merging. The PR is here: https://github.com/nextcloud/mail/pull/298